aspnetcore/fundamentals/minimal-apis/security.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Configuring authentication for minimal APIs
+author: captainsafia
+description: Learn how to configure authentication and authorization in minimal API apps
+ms.author: safia
+monikerRange: '>= aspnetcore-7.0'
```

Comment:
What is specific to .NET 7? Is it worth adding some conditional tags so this doc is visible with .NET 6?

Reply:
The flow documented here (like being able to read options from config and the user-jwts tool) are .NET 7-specific.